doc/manual.md

```diff
@@ -3060,6 +3060,30 @@ But these will compile successfully:
   doAssert string(s) == "test"
   ```
 
+If a procedure call returns a `var`-typed value and is assigned to a `var`-declared variable, it is copied. This way, you can modify any `var`-declared variable and be sure it will not affect variables out of its scope.
+
+By contrast, passing a `var`-typed return value to another procedure call does not copy it. See [Var-parameters](#procedures-var-parameters)
+
+For a `var`-declared variable that *can* modify an existing `var`-typed value, the `addr` procedure can be used to create a reference type, see [Reference and pointer types](#types-reference-and-pointer-types)
+
+
+```nim
+var foo = Foo(x: "this")
+
+proc getX(f: var Foo): var string
```

Comment:
This is not valid code ?, `=` missing.

I would add a `type Foo` for completeness.

Reply:
Fixed! Thanks